builds/build-release.yml

```diff
@@ -1,33 +1,27 @@
+# This pipeline is manually triggered
 trigger: none
 
 pr: none
 
-schedules:
-  - cron: "0 0 * * *"
-    displayName: Mon-Fri at Midnight
-    branches:
-      include:
-        - main
-    always: true
```

Comment:
Why is the schedule removed?

Reply:
I don't see the benefit to running this every night, I could be convinced though 😄

Reply:
Nah, that's all good.  Is it now just on-demand, or is there a new trigger?

Reply:
Yeah just run manually now, the build number will still be incremented every time